acdc/greaterthan/utils.py

```diff
@@ -1,5 +1,19 @@
-#%%
-
+from collections import OrderedDict
```

Comment:
We don't need all these imports right? espeically click, it's a new package

Reply:
That is correct